recipes/cpp-subprocess/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conans import ConanFile, tools
+
+
+class CppSubprocess(ConanFile):
+    name = "cpp-subprocess"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nextsilicon/cpp-subprocess#readme"
+    topics = ("subprocess", "os", "fork")
+    description = ("Subprocessing with modern C++, "
+                   "The only goal was to develop something that is as close as"
+                   "python subprocess module in dealing with processes.")
+    # No settings/options are necessary, this is header only
+    exports_sources = "subprocess.hpp"
+    no_copy_source = True
+
+    _source_subfolder = 'cpp-subprocess'
+
+    def source(self):
+        tools.get(**self.conan_data['sources'][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def package(self):
+        self.copy("subprocess.hpp", dst="include", src=self._source_subfolder)
```

Comment:
```suggestion
        self.copy("subprocess.hpp", dst="include", src=self._source_subfolder)
```

Should be namespaced: https://github.com/arun11299/cpp-subprocess/blob/fa58db9d6e250168b864970051b07a962b67d915/CMakeLists.txt#L12

Reply:
Not entirely sure I understand.

Packaging it to `include` allows you to simply `#include "subprocess.hpp"`, doesn't it?

Reply:
Yes but the install target of upstream CMakeLists suggests that it should be installed under subprocess folder, and therefore included with `#include <subprocess/subprocess.hpp>`. But the README is not consistent, so I don't know.

Reply:
Will fixing the README and including it through `#include <cpp-subprocess/subprocess.h>` make sense?

If it does let's fix it that way. Actually taking the global `subprocess.hpp` namespace sounds like a bad idea to me.

Reply:
namespaced it under `cpp-subprocess/subprocess.hpp`, and updated the README and CMakeLists.txt.